### PR TITLE
[RA1] Add Barbican API

### LINKED ISSUE #1076: [RA1]Clarify SFC, Cyborg, Barbican or Telemetry API
A few "should" requirements obscure the common CNTT API and the related testing (RC).
I would rather suggest "must" or no requirement for the next services (SFC, Cyborg, Barbican, Telemetry, etc.):
- req.asr.mon.01
- req.inf.ntw.04
- req.sec.zon.02

In this state, I would have considered that we do complete the API (major/minimal versions and capabilities) and we do add the related testing in IaaS verification (most of them are already covered). But at the end of the day the testing shouldn't be part of RC as they are optional.

### PR DESCRIPTION
**Pull request by collivier**
It also fixes a few nits to unify the tabs.

Closes #1076

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

**Comment by rabiabdel** [comment]
@pgoyal01 please review/approve

File: doc/ref_arch/openstack/chapters/chapter05.md [hunk 1]
```diff
@@ -294,13 +294,21 @@ The Libvirt APIs are documented in https://libvirt.org/html/index.html.
...
+| Accelerator: Cyborg   | v2              |
 
 Acceleration Service API: https://docs.openstack.org/api-ref/accelerator/v2/index.html
 Please note that the initial version of the [Cyborg API v1.0](https://docs.openstack.org/cyborg/stein/admin/api.html) was deprecated in the OpenStack Train release and will be removed in the Ussuri release.
 
+### 5.3.5. Barbican
+
+| **OpenStack Service**           | **API Version** |
+|---------------------------------|-----------------|
+| OpenStack Key Manager: Barbican | v1              |
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter05.md]: @collivier to be consistent can you delete "OpenStack" from "OpenStack Key Manager"? Thanks

**Comment by rabiabdel** [comment]
@collivier are you OK with the change suggested by @pgoyal01 ?

File: doc/ref_arch/openstack/chapters/chapter05.md (hunk 1 above)
Comment (rabi-abdel) [reply doc/ref_arch/openstack/chapters/chapter05.md]: fixed in https://github.com/cntt-n/CNTT/pull/1357/commits/ac76bcbcb152d1952127452b7475d5ed2826292d

**Pull request merged**